.github/workflows/py-build-pr.yml

```diff
@@ -6,15 +6,15 @@ jobs:
     strategy:
       max-parallel: 6
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
Wanna drop 3.7?

Reply:
I do want to drop it but can we?

Reply:
There haven't been API changes in a while to the library so 3.7 users wouldn't be "missing anything." / I don't think we plan to have any major changes soon so that is also extended. From pypistats it looks like 3.7 is consistantly our lowest download portion of all 3 versions:

![image](https://user-images.githubusercontent.com/17132317/151242378-c06470a4-de42-41a9-873b-c8e226d969f0.png)

Reply:
Also, pandas just dropped py37 from their latest release.

Reply:
I don't understand why you say 3.7 users wouldn't be missing anything if we dropped build support for it.

Reply:
Yea that doesn't really make sense in hindsight. But the fact that pandas dropped support for it to me means we can also move on :shrug:

Reply:
I'll have to see how many workflows we have internally that are currently on 3.7 :)